VirtoCommerce.CatalogModule.Web/Scripts/blades/images-add.tpl.html

```diff
@@ -52,7 +52,10 @@
                 </div>
 
                 <div class="form-input __file">
-                    <input ng-model="blade.newExternalImageUrl" type="url" placeholder="{{ 'catalog.blades.images.placeholders.image-url' | translate }}" tabindex="-1">
+                    <input ng-model="blade.newExternalImageUrl" type="url" placeholder="{{ 'catalog.blades.images.placeholders.image-url' | translate }}" tabindex="-1" style="padding-right: 80px">
```

Comment:
The not strict comment, but in the future need to avoid of inline styles usage and use css classes instead